src/Description.js

```diff
@@ -73,6 +74,9 @@ export default function Description() {
         </Col>
         <Col md='auto'></Col>
       </Row>
+      <Row>
```

Comment:
I think this is causing a weird effect on the parent (outside of website content) border. If a col className=mb2 is added here to wrap the Footer component, the rounded border background will have some margin between it and the parent background and border.

Reply:
I think we also need to add COL before and after the Footer component COL so the design and responsiveness (of the page to sizing) will be consistent.